Jenkinsfile

```diff
@@ -10,29 +12,37 @@ node {
         checkout scm
       }
       updateGithubCommitStatus('PENDING', "${env.WORKSPACE}/src")
+
       stage('dep') {
         godep()
       }
+
       stage('Build') {
         parallel (
           "go lint": {
             golint()
           },
-          "go build": {
-            build()
+          "docker build": {
+            image = docker(repo)
+          },
+          "deb package": {
+            build(repo)
+            gitPbuilder('bionic')
           }
         )
       }
```

Comment:
so the parallel stages could be:
- lint + build
- docker + deb
- registry + aptly

Reply:
Yes but if that's OK with you I have lost enough time on this trivial change for the moment, could this be addressed later?

Reply:
On the other hand, you've learned a ton.

Reply:
Agreed ;)